contracts/README.md

```diff
@@ -0,0 +1,55 @@
+# Finschia Smart contract Samples
+
+Collection of samples to help you develop smart contracts on Finschia.
+
+## Samples
+
+### Introduction to CosmWasm
+* [Hello World](https://github.com/athena-consulting/cosmwasm-by-example/tree/main/hello-world): This smart contract provides an implementation for querying a "Hello World" message.
+* [Counter](https://github.com/athena-consulting/cosmwasm-by-example/tree/main/counter): Counter base smart contract that allows for counter increment and reset.
+* [Primitives](https://github.com/athena-consulting/cosmwasm-by-example/tree/main/primitives): Early introduction to some primitive variables in CosmWasm and how they work.
+* [Variables in CosmWasm](https://github.com/athena-consulting/cosmwasm-by-example/tree/main/variables): An overview of state and global variables within CosmWasm and methods to access them.
+* [Smart Contract Initialization](https://github.com/athena-consulting/cosmwasm-by-example/tree/main/instantiation): This document elucidates the instantiation process of a smart contract. It discusses the InstantiateMsg structure passed during contract creation and the instantiate function that runs upon contract execution.
+* [Send Native Tokens](https://github.com/athena-consulting/cosmwasm-by-example/tree/main/send-tokens): This document outlines a smart contract designed to send a blockchain's native tokens to a recipient specified by the original sender in the execute message.
+* [Read-Write State](https://github.com/athena-consulting/cosmwasm-by-example/tree/main/read-write-state): Smart contract that explains the basics of reading and writing to the state in a smart contract.
+* [Responses and Attributes in CosmWasm](https://github.com/athena-consulting/cosmwasm-by-example/tree/main/responses-attributes): Smart contract that explains the basics of responses and attributes. 
+* [CosmWasm Maths Operations](https://github.com/athena-consulting/cosmwasm-by-example/tree/main/cosmwasm-math): This is a example of a CosmWasm contract that implements simple maths operations like addition, substraction, multiplication, division, modulo and exponential.
+* [Receive cw20 Tokens in Your Contract](https://github.com/athena-consulting/cosmwasm-by-example/tree/main/recieving-cw20-tokens): An example of a CosmWasm contract that implements the cw20 Receiver Interface.
+
+### Application Samples
+* [Timelock](https://github.com/athena-consulting/cosmwasm-by-example/tree/main/timelock): Timelock is a smart contract that introduces a delay mechanism for executing function calls on other smart contracts.
+* [English-Auction Contract](https://github.com/athena-consulting/cosmwasm-by-example/tree/main/dutch-auction): The Dutch Auction Contract is a smart contract implementation that enables the creation and execution of Dutch Auctions. 
```

Comment:
Thie link and description is saying dutch action not english action. Which one is right?

Reply:
please check it again. It is `Dutch auction`. 
https://en.wikipedia.org/wiki/Dutch_auction

Reply:
Sorry. The title and content are different. I'll check it out.